docs/about/index.rst

```diff
@@ -17,13 +40,156 @@ Code of conduct
+:ref:`reach out <communication>`.  The inclusion of tasks in the list below
+does not mean that we are committing to deliver them by a specific date.
+It merely represents a current list of tasks that members of the community
+work on. If you are interested to work on a topic that is not covered, please
+reach out.
+
+Tasks
+~~~~~
+
+#. *Improve the `fairlearn.metrics` API*
+   More information in the
+   `fairlearn-proposals <https://github.com/fairlearn/fairlearn-proposals/blob/master/api/METRICS.md>`_
+   repository.
+
+   **How you can help:** critique the proposal, suggest new metrics, work on
+   implementing new metrics.
+
+#. *Improve the project documentation*
+   The Fairlearn website is still very new and raw with lots of gaps.
+   `This proposal <https://github.com/fairlearn/fairlearn-proposals/pull/8>`_
+   outlines the planned efforts, some of which have been completed. There are
+   still massive gaps, particularly in the user guides.
+
+   **How you can help:** critique the current content, improve writing and
+   examples, port the current documentation to the numpydoc format.
+
+#. *Improve example notebooks and add new ones*
+   The repository started out with all notebooks in the `notebooks` directory
+   as `.ipynb` files. These notebooks are now being
+   converted into `.py` files that
+   `render properly on the website <https://fairlearn.github.io/auto_examples/notebooks/index.html>`_,
+   while still being downloadable as `.py` or `.ipynb` files.
+   
+   - Some of the notebooks may be better off as parts of the user guide.
+   - As pointed out in various issues the existing notebooks have plenty of
+     potential for improvement to better reflect the sociotechnical focus of Fairlearn.
+     To accelerate this process, we experiment with weekly sessions
+     where people can discuss individual example notebooks in detail.
+   - There are only very few existing example notebooks. To cover a
+     wider range of applications we need to work with stakeholders to develop
+     more guides.
+
+   **How you can help:** critique and improve existing notebooks, join the
+   discussion sessions, create new notebooks, move the notebooks to the user
+   guide.
+
+#. *Nurture a diverse community of contributors*
+   We would like to ensure that all feel welcome and can contribute in a way
+   of their choosing. So far we have monthly developer calls.  We are
+   experimenting with adding deep dive sessions around sociotechnical aspects
+   of notebooks.  We also work with universities on engaging with student
+   contributors through course projects.  Finally, we are working to improve
+   documentation.
+   
+   **How you can help:** reach out with feedback on what is working and what
+   is not working; suggest how to improve things; point out where
+   documentation, our processes or any other aspect of the projects create
+   barriers of entry.
+
+#. *Move towards compatibility with scikit-learn*
```

Comment:
Might want to note that while we're aiming for compatibility with scikit-learn, we do not want to restrict ourselves to it.